samples/GraphQL.DataLoader.Example/DataLoaderGql.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+    <PropertyGroup>
+        <TargetFramework>net8.0</TargetFramework>
+        <Nullable>enable</Nullable>
+        <ImplicitUsings>enable</ImplicitUsings>
+      <IsPackable>false</IsPackable>
+      <IsTestProject>false</IsTestProject>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="GraphQL" Version="8.2.1" />
+        <PackageReference Include="GraphQL.DataLoader" Version="8.2.1" />
+        <PackageReference Include="GraphQL.MicrosoftDI" Version="8.2.1" />
+        <PackageReference Include="GraphQL.Server.All" Version="8.1.0" />
+        <PackageReference Include="GraphQL.Server.Ui.Altair" Version="8.1.0" />
+        <PackageReference Include="GraphQL.SystemTextJson" Version="8.2.1" />
```

Comment:
Needs to be ProjectReference not PackageReference

Reply:
this actually seems problematic, as the Server Package already imports these Dependencies as Nugets and is not available here for Project Reference.

Reply:
Right, that is why I suggested we might make a very-simple server project replacement ("polyfill") so that the sample is representative.  Or we have to move the sample out of the repo.  But it defeats the purpose to have the sample here if changes to the codebase are not reflected in the execution of the sample project.

Reply:
i will look to do this today. so what were you thinking? Just Copy the Code from server Project and make clear it's a pollyfill by naming and Project Config? Or would it be needed to be trimmed down a bunch?

Reply:
Look in the Harness sample and see the middleware file. It’s very simple. Use that along with an extension to link up UseGraphQL to the middleware so that the same code in the sample projects can be used with the server project.

Reply:
Have a look. i stole the minimum part from Harness Project and added an extra Comment in Program.cs.
Still works as expected locally

Reply:
Here, I've extracted it to a separate project: https://github.com/graphql-dotnet/graphql-dotnet/pull/4102

Reply:
that's a really nice Idea and can help making future example Projects simpler to get up and running.

Reply:
ok, shouldn't i first switch to using #4102 first?